extensions/extensions-helm/aissemble-spark-history-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if and .Values.enable .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}-ingress
+  {{- with .Values.ingress.metadata.annotations }}
+  annotations:
+  {{- toYaml . | nindent 8 }}
+  {{- end }}
+spec:
+  ingressClassName: {{ .Values.ingress.ingressClassName }}
+  rules:
+    {{- range .Values.ingress.hosts }}
+    - http:
```

Comment:
Don't you need the `host` attribute?

Reply:
I don't THINK so-- Why?

Reply:
We have it in all our other ingress. Thought that was how you define the ingress URL?

Reply:
Ah, I see it now.  Added.